raster/src/main/scala/geotrellis/raster/crop/CropMethods.scala

```diff
@@ -0,0 +1,35 @@
+package geotrellis.raster.crop
+
+import geotrellis.vector._
+import geotrellis.raster._
+
+trait CropMethods[T] extends MethodExtensions[T] {
```

Comment:
The best thing about `MethodExtensions[T]` is that it gives something to search for and tags the trait with a noun.